src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (...args) => {
```

Comment:
The use of the rest parameter '...args' is not necessary since the 'adder' function is intended to be called with only one argument at a time (except for the final call, which has no arguments). Consider simplifying the function to accept a single argument.